multiqc/utils/search_patterns.yaml

```
@@ -279,6 +279,10 @@ gatk/varianteval:
 gatk/base_recalibrator:
     contents: '#:GATKTable:Arguments:Recalibration'
     num_lines: 3
+gffcompare:
+    fn: '*.stats'
+    contents: '# gffcompare'
+    num_lines: 2
```

```suggestion
    num_lines: 10
```
Should be a little more generous just in case there is random stuff prepended to the log.

---

No need as not stdout / stderr